spec/std/isa/csr/V/vcsr.yaml

```
@@ -0,0 +1,34 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: vcsr
+long_name: Vector Control and Status Register
+address: 0x00F
+writable: true
+priv_mode: U
+length: MXLEN
+description: Allows access to vxrm and vxsat CSRs
```

"Allows access" sound like an enable bit, so I suggest
```suggestion
description: Contains aliases to vxrm and vxsat CSRs
```